docs/workforce-management-framework/topics/con-passportauth-module.adoc

```diff
@@ -1,9 +1,11 @@
 [id='con-passportauth-module-{chapter}']
 = Introducing the {WFM-RC-NameShort} PassportAuth Module
 
-The link:{WFM-RC-CoreTreeURL}{WFM-RC-Branch}/cloud/passportauth[PassportAuth] module is {WFM-RC-NameShort}'s implementation of *Passport.js* and is the default authentication and authorization module for {WFM-RC-NameShort}.
-*PassportAuth* provides pluggable authentication using link:http://passportjs.org/docs/configure[strategies].
-*PassportAuth* offers a comprehensive set of strategies that support authentication using a username and password, Facebook logon, Twitter logon, and more.
+The link:{WFM-RC-CoreTreeURL}{WFM-RC-Branch}/cloud/passportauth[PassportAuth] module is {WFM-RC-NameShort}'s implementation of *Passport.js*.
+It is the default authentication and authorization module for {WFM-RC-NameShort}.
+
+*passport.js* provides pluggable authentication using link:http://passportjs.org/docs/configure[strategies].
```

Comment:
This has changed from *PassportAuth* to *passport.js*. This doesn't make sense to me as this module is introducing the PassportAuth Module. This should be  *PassportAuth* or else we remove this line and the line below as I do not want to confused the reader (as I was).